App/src/main/java/vut/fit/gja2023/app/controller/MainController.java

```diff
@@ -1,13 +1,22 @@
 package vut.fit.gja2023.app.controller;
 
 import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestMapping;
+import vut.fit.gja2023.app.service.SystemManagerService;
 
 @Controller
 @RequiredArgsConstructor
 public class MainController {
 
+    SystemManagerService systemManagerService;
+
+    @Autowired
+    public MainController(SystemManagerService systemManagerService) {
+        this.systemManagerService = systemManagerService;
+    }
```

Comment:
nvm ako sa to robi v kontroleroch, ale nestaci dat a nechceme mat `systemManagerService` private final, a teda potom toto spravi `@RequiredArgsConstructor`?